Client/Frontend/Browser/QRCodeViewController.swift

```diff
@@ -15,8 +15,8 @@ protocol QRCodeViewControllerDelegate: AnyObject {
 class QRCodeViewController: UIViewController {
```

Comment:
My comments mean this class would need to be [`Themeable`](https://github.com/mozilla-mobile/firefox-ios/wiki/Theming-system) following that protocol.